src/main/java/com/ngs/stash/externalhooks/hook/ExternalMergeCheckHook.java

```diff
@@ -137,44 +127,34 @@ public void check(
         String summaryMsg = "Merge request failed";
 
         try {
-            int Result = runExternalHooks(pb, refChanges, hookResponse);
-            if (Result != 0) {
-                String rawDetailedMsg = errWriter.toString();
-                String prePrefix = "<pre style=\"overflow: auto; white-space: nowrap;\">";
-                String preSuffix = "</pre>";
-                String detailedMsg = prePrefix + rawDetailedMsg.replaceAll("(\r\n|\n)", "<br/>").replaceAll(" ", " ") + preSuffix;
```

Comment:
Does dropping this `detailedMsg` formatting affect how this displays in the UI when the check fails? Is that desirable? (Or perhaps the new hook SPI doesn't allow for the formatting to work correctly, anymore? I could believe that.)